src/main/java/org/craftercms/studio/impl/v2/service/marketplace/MarketplaceServiceImpl.java

```diff
@@ -93,4 +93,15 @@ public void copyPlugin(String siteId, @ValidateSecurePathParam(name = "path") St
         marketplaceServiceInternal.copyPlugin(siteId, path);
     }
 
+    @Override
+    @HasPermission(type = DefaultPermission.class, action = PERMISSION_INSTALL_PLUGINS)
```

Comment:
Should this be updated to `removePlugin` permission?

Reply:
same as https://github.com/craftercms/studio/pull/2770#discussion_r728460885

Reply:
Let's add it. See my note earlier.